dictionary.cc

```diff
@@ -171,6 +171,11 @@ vector< wstring > Class::getAlternateWritings( wstring const & )
   return vector< wstring >();
 }
 
+QString Class::getContainingFolder()
+{
+  return QFileInfo( QString::fromStdString( dictionaryFiles[ 0 ] ) ).absolutePath();
+}
+
```

Comment:
is the returned value a file or a folder ?
dirname should be the parent directory of the file.

Reply:
It is the folder

For example, `/home/a/nice.mdx` will return `/home/a`